src/components/Semester/SemesterView.vue

```diff
@@ -26,18 +26,19 @@
         data-disable-interaction="1"
         data-step="4"
         data-tooltipClass="tooltipCenter"
+        v-if="!isMobile"
```

Comment:
I believe we brought back the ability to toggle between compact and not compact view in mobile so you can remove this line